packages/shared/lib/utils.ts

```diff
@@ -126,7 +126,7 @@ export function debounce(callback: () => void, wait = 500): (...args: unknown[])
 /**
  * Set text to clipboard
  */
-export const setClipboard = (input: string): boolean => {
+export const setClipboard = (input: string, shouldNotify: boolean = true): boolean => {
```

Comment:
Can we create regular functions everywhere? :D

Reply:
yeah, this is existing code and I amended the param